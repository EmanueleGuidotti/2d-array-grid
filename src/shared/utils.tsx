export const grid5 = [
	[0, 0, 0, 0, 1],
	[1, 1, 0, 0, 0],
	[1, 1, 0, 1, 1],
	[0, 0, 0, 0, 0],
	[1, 1, 1, 0, 0]
];

export const grid3 = [
    [0, 0, 0],
    [1, 1, 0],
    [1, 1, 0],
];