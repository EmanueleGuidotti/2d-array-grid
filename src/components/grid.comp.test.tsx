import React from 'react';
import GridComponent from './grid.comp';
import renderer from 'react-test-renderer';
import { grid5, grid3 } from '../shared/data';

test('Expect render 5 row x 5 cells component', () => {
	const data: number[][] = grid5;
	const cmp = renderer
		.create(<GridComponent data={grid5}></GridComponent>)
		.toJSON();
	expect(cmp).toMatchSnapshot();
});

test('Expect render 3 row x 3 cells component', () => {
    const data: number[][] = grid3;
    const cmp = renderer
        .create(<GridComponent data={grid3}></GridComponent>)
        .toJSON();
    expect(cmp).toMatchSnapshot();
});
