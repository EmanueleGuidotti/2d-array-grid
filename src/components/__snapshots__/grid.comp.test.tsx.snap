// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect render 3 row x 3 cells component 1`] = `
<View
  style={
    Object {
      "flexDirection": "column",
      "flexWrap": "wrap",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "stretch",
        "display": "flex",
        "flexDirection": "row",
        "flexWrap": "nowrap",
        "justifyContent": "space-around",
        "width": 750,
      }
    }
  >
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderColor": "#cccccc",
          "borderStyle": "solid",
          "borderWidth": 1,
          "flexBasis": "auto",
          "flexGrow": 1,
          "flexShrink": 1,
          "height": 251,
          "margin": 0,
          "opacity": 1,
          "width": 251,
        }
      }
    />
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderColor": "#cccccc",
          "borderStyle": "solid",
          "borderWidth": 1,
          "flexBasis": "auto",
          "flexGrow": 1,
          "flexShrink": 1,
          "height": 251,
          "margin": 0,
          "opacity": 1,
          "width": 251,
        }
      }
    />
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderColor": "#cccccc",
          "borderStyle": "solid",
          "borderWidth": 1,
          "flexBasis": "auto",
          "flexGrow": 1,
          "flexShrink": 1,
          "height": 251,
          "margin": 0,
          "opacity": 1,
          "width": 251,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "alignItems": "stretch",
        "display": "flex",
        "flexDirection": "row",
        "flexWrap": "nowrap",
        "justifyContent": "space-around",
        "width": 750,
      }
    }
  >
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#000000",
          "borderColor": "#cccccc",
          "borderStyle": "solid",
          "borderWidth": 1,
          "flexBasis": "auto",
          "flexGrow": 1,
          "flexShrink": 1,
          "height": 251,
          "margin": 0,
          "opacity": 1,
          "width": 251,
        }
      }
    />
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#000000",
          "borderColor": "#cccccc",
          "borderStyle": "solid",
          "borderWidth": 1,
          "flexBasis": "auto",
          "flexGrow": 1,
          "flexShrink": 1,
          "height": 251,
          "margin": 0,
          "opacity": 1,
          "width": 251,
        }
      }
    />
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderColor": "#cccccc",
          "borderStyle": "solid",
          "borderWidth": 1,
          "flexBasis": "auto",
          "flexGrow": 1,
          "flexShrink": 1,
          "height": 251,
          "margin": 0,
          "opacity": 1,
          "width": 251,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "alignItems": "stretch",
        "display": "flex",
        "flexDirection": "row",
        "flexWrap": "nowrap",
        "justifyContent": "space-around",
        "width": 750,
      }
    }
  >
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#000000",
          "borderColor": "#cccccc",
          "borderStyle": "solid",
          "borderWidth": 1,
          "flexBasis": "auto",
          "flexGrow": 1,
          "flexShrink": 1,
          "height": 251,
          "margin": 0,
          "opacity": 1,
          "width": 251,
        }
      }
    />
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#000000",
          "borderColor": "#cccccc",
          "borderStyle": "solid",
          "borderWidth": 1,
          "flexBasis": "auto",
          "flexGrow": 1,
          "flexShrink": 1,
          "height": 251,
          "margin": 0,
          "opacity": 1,
          "width": 251,
        }
      }
    />
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderColor": "#cccccc",
          "borderStyle": "solid",
          "borderWidth": 1,
          "flexBasis": "auto",
          "flexGrow": 1,
          "flexShrink": 1,
          "height": 251,
          "margin": 0,
          "opacity": 1,
          "width": 251,
        }
      }
    />
  </View>
</View>
`;

exports[`Expect render 5 row x 5 cells component 1`] = `
<View
  style={
    Object {
      "flexDirection": "column",
      "flexWrap": "wrap",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "stretch",
        "display": "flex",
        "flexDirection": "row",
        "flexWrap": "nowrap",
        "justifyContent": "space-around",
        "width": 750,
      }
    }
  >
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderColor": "#cccccc",
          "borderStyle": "solid",
          "borderWidth": 1,
          "flexBasis": "auto",
          "flexGrow": 1,
          "flexShrink": 1,
          "height": 151,
          "margin": 0,
          "opacity": 1,
          "width": 151,
        }
      }
    />
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderColor": "#cccccc",
          "borderStyle": "solid",
          "borderWidth": 1,
          "flexBasis": "auto",
          "flexGrow": 1,
          "flexShrink": 1,
          "height": 151,
          "margin": 0,
          "opacity": 1,
          "width": 151,
        }
      }
    />
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderColor": "#cccccc",
          "borderStyle": "solid",
          "borderWidth": 1,
          "flexBasis": "auto",
          "flexGrow": 1,
          "flexShrink": 1,
          "height": 151,
          "margin": 0,
          "opacity": 1,
          "width": 151,
        }
      }
    />
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderColor": "#cccccc",
          "borderStyle": "solid",
          "borderWidth": 1,
          "flexBasis": "auto",
          "flexGrow": 1,
          "flexShrink": 1,
          "height": 151,
          "margin": 0,
          "opacity": 1,
          "width": 151,
        }
      }
    />
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#000000",
          "borderColor": "#cccccc",
          "borderStyle": "solid",
          "borderWidth": 1,
          "flexBasis": "auto",
          "flexGrow": 1,
          "flexShrink": 1,
          "height": 151,
          "margin": 0,
          "opacity": 1,
          "width": 151,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "alignItems": "stretch",
        "display": "flex",
        "flexDirection": "row",
        "flexWrap": "nowrap",
        "justifyContent": "space-around",
        "width": 750,
      }
    }
  >
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#000000",
          "borderColor": "#cccccc",
          "borderStyle": "solid",
          "borderWidth": 1,
          "flexBasis": "auto",
          "flexGrow": 1,
          "flexShrink": 1,
          "height": 151,
          "margin": 0,
          "opacity": 1,
          "width": 151,
        }
      }
    />
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#000000",
          "borderColor": "#cccccc",
          "borderStyle": "solid",
          "borderWidth": 1,
          "flexBasis": "auto",
          "flexGrow": 1,
          "flexShrink": 1,
          "height": 151,
          "margin": 0,
          "opacity": 1,
          "width": 151,
        }
      }
    />
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderColor": "#cccccc",
          "borderStyle": "solid",
          "borderWidth": 1,
          "flexBasis": "auto",
          "flexGrow": 1,
          "flexShrink": 1,
          "height": 151,
          "margin": 0,
          "opacity": 1,
          "width": 151,
        }
      }
    />
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderColor": "#cccccc",
          "borderStyle": "solid",
          "borderWidth": 1,
          "flexBasis": "auto",
          "flexGrow": 1,
          "flexShrink": 1,
          "height": 151,
          "margin": 0,
          "opacity": 1,
          "width": 151,
        }
      }
    />
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderColor": "#cccccc",
          "borderStyle": "solid",
          "borderWidth": 1,
          "flexBasis": "auto",
          "flexGrow": 1,
          "flexShrink": 1,
          "height": 151,
          "margin": 0,
          "opacity": 1,
          "width": 151,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "alignItems": "stretch",
        "display": "flex",
        "flexDirection": "row",
        "flexWrap": "nowrap",
        "justifyContent": "space-around",
        "width": 750,
      }
    }
  >
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#000000",
          "borderColor": "#cccccc",
          "borderStyle": "solid",
          "borderWidth": 1,
          "flexBasis": "auto",
          "flexGrow": 1,
          "flexShrink": 1,
          "height": 151,
          "margin": 0,
          "opacity": 1,
          "width": 151,
        }
      }
    />
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#000000",
          "borderColor": "#cccccc",
          "borderStyle": "solid",
          "borderWidth": 1,
          "flexBasis": "auto",
          "flexGrow": 1,
          "flexShrink": 1,
          "height": 151,
          "margin": 0,
          "opacity": 1,
          "width": 151,
        }
      }
    />
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderColor": "#cccccc",
          "borderStyle": "solid",
          "borderWidth": 1,
          "flexBasis": "auto",
          "flexGrow": 1,
          "flexShrink": 1,
          "height": 151,
          "margin": 0,
          "opacity": 1,
          "width": 151,
        }
      }
    />
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#000000",
          "borderColor": "#cccccc",
          "borderStyle": "solid",
          "borderWidth": 1,
          "flexBasis": "auto",
          "flexGrow": 1,
          "flexShrink": 1,
          "height": 151,
          "margin": 0,
          "opacity": 1,
          "width": 151,
        }
      }
    />
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#000000",
          "borderColor": "#cccccc",
          "borderStyle": "solid",
          "borderWidth": 1,
          "flexBasis": "auto",
          "flexGrow": 1,
          "flexShrink": 1,
          "height": 151,
          "margin": 0,
          "opacity": 1,
          "width": 151,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "alignItems": "stretch",
        "display": "flex",
        "flexDirection": "row",
        "flexWrap": "nowrap",
        "justifyContent": "space-around",
        "width": 750,
      }
    }
  >
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderColor": "#cccccc",
          "borderStyle": "solid",
          "borderWidth": 1,
          "flexBasis": "auto",
          "flexGrow": 1,
          "flexShrink": 1,
          "height": 151,
          "margin": 0,
          "opacity": 1,
          "width": 151,
        }
      }
    />
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderColor": "#cccccc",
          "borderStyle": "solid",
          "borderWidth": 1,
          "flexBasis": "auto",
          "flexGrow": 1,
          "flexShrink": 1,
          "height": 151,
          "margin": 0,
          "opacity": 1,
          "width": 151,
        }
      }
    />
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderColor": "#cccccc",
          "borderStyle": "solid",
          "borderWidth": 1,
          "flexBasis": "auto",
          "flexGrow": 1,
          "flexShrink": 1,
          "height": 151,
          "margin": 0,
          "opacity": 1,
          "width": 151,
        }
      }
    />
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderColor": "#cccccc",
          "borderStyle": "solid",
          "borderWidth": 1,
          "flexBasis": "auto",
          "flexGrow": 1,
          "flexShrink": 1,
          "height": 151,
          "margin": 0,
          "opacity": 1,
          "width": 151,
        }
      }
    />
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderColor": "#cccccc",
          "borderStyle": "solid",
          "borderWidth": 1,
          "flexBasis": "auto",
          "flexGrow": 1,
          "flexShrink": 1,
          "height": 151,
          "margin": 0,
          "opacity": 1,
          "width": 151,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "alignItems": "stretch",
        "display": "flex",
        "flexDirection": "row",
        "flexWrap": "nowrap",
        "justifyContent": "space-around",
        "width": 750,
      }
    }
  >
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#000000",
          "borderColor": "#cccccc",
          "borderStyle": "solid",
          "borderWidth": 1,
          "flexBasis": "auto",
          "flexGrow": 1,
          "flexShrink": 1,
          "height": 151,
          "margin": 0,
          "opacity": 1,
          "width": 151,
        }
      }
    />
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#000000",
          "borderColor": "#cccccc",
          "borderStyle": "solid",
          "borderWidth": 1,
          "flexBasis": "auto",
          "flexGrow": 1,
          "flexShrink": 1,
          "height": 151,
          "margin": 0,
          "opacity": 1,
          "width": 151,
        }
      }
    />
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#000000",
          "borderColor": "#cccccc",
          "borderStyle": "solid",
          "borderWidth": 1,
          "flexBasis": "auto",
          "flexGrow": 1,
          "flexShrink": 1,
          "height": 151,
          "margin": 0,
          "opacity": 1,
          "width": 151,
        }
      }
    />
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderColor": "#cccccc",
          "borderStyle": "solid",
          "borderWidth": 1,
          "flexBasis": "auto",
          "flexGrow": 1,
          "flexShrink": 1,
          "height": 151,
          "margin": 0,
          "opacity": 1,
          "width": 151,
        }
      }
    />
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderColor": "#cccccc",
          "borderStyle": "solid",
          "borderWidth": 1,
          "flexBasis": "auto",
          "flexGrow": 1,
          "flexShrink": 1,
          "height": 151,
          "margin": 0,
          "opacity": 1,
          "width": 151,
        }
      }
    />
  </View>
</View>
`;
